OrbitService/CaptureServiceImpl.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include "CaptureServiceImpl.h"
 
-#include "CaptureResponseListener.h"
```

Comment:
The idea behind this interface was that we can have a second implementation for SaveToFileResponseListener, now the Buffer implemenation has gprc-spcific logic as well as bufferring logic, maybe keep them separate? have the buffer and delegate sending to this listener?

Reply:
Ok, I see. The way I was thinking it buffering and sending are tied together here as buffering is done in order to send fewer messages, but to save to file something different could be done, e.g., writing event by event as maybe the buffering is done by the os. That's why `CaptureEventBuffer` (but maybe the name should be different) only has `AddEvent`, and an additional `SaveToFile...` could have a different buffering logic. But this is unless you want to have the same way of buffering and be able to only swap sending on gRPC with saving to file.
So in other words if you think that later only the "sending" part should be swappable (with a fixed buffering strategy) I will separate the two interfaces, otherwise if you think that it is ok the way it is then I'll keep them together. Up to you.

Reply:
Yep, I think buffering strategy with threading should be separate from sending/saving strategy and will likely be reused for savetofile

Reply:
Ok, done, please re-review.